platform/macos/os_macos.mm

```diff
@@ -341,11 +341,11 @@
 
 	String font_name = p_font_name;
 	if (font_name.to_lower() == "sans-serif") {
-		font_name = "Helvetica";
+		font_name = "SF Pro";
```

Comment:
```suggestion
		CTFontRef ctfont = CTFontCreateUIFontForLanguage(kCTFontUIFontSystem, 0.0, nullptr);
		CTFontDescriptorRef font = CTFontCopyFontDescriptor(ctfont);
		if (font) {
			CFURLRef url = (CFURLRef)CTFontDescriptorCopyAttribute(font, kCTFontURLAttribute);
			if (url) {
				NSString *font_path = [NSString stringWithString:[(__bridge NSURL *)url path]];
				ret = String::utf8([font_path UTF8String]);
				CFRelease(url);
			}
			CFRelease(font);
		}
		CFRelease(ctfont);
		return ret;
```

Probably the correct way to get the default system UI font is doing something like this, instead of looking it up by name.